image/image.go

```diff
@@ -301,6 +301,13 @@ func createBundle(w walker, m *manifest, dest, rootfs string) error {
 			if err2 := os.MkdirAll(dest, 0755); err2 != nil {
 				return err2
 			}
+			defer func() {
+				if err != nil {
+					if err3 := os.RemoveAll(dest); err3 != nil {
+						fmt.Printf("warning: failed to clean up %q\n", dest)
```

Comment:
> Let's print it here, updated

Did you forget to push?  I still see this PR pointing at c5ed3a8ec, has a 2017-08-04 commit date.